src/yaml.jl

```diff
@@ -1,26 +1,26 @@
 """
-    dftramo_run(filename::AbstractString, software::AbstractString="vasp")
+    dftramo_run(filename::AbstractString)
 
 Automatically runs DFTraMO from a configuration yaml file.
 """
-function dftramo_run(filename::AbstractString, software::AbstractString="vasp")
-    (runs, checkpoint, auto_psphere, dftinfo, emin, emax) = read_run_yaml(filename, software)
-    occ_states = get_occupied_states(dftinfo.wave, emin, emax)
-    super = Supercell(dftinfo.xtal, ORB_DICT)
+function dftramo_run(filename::AbstractString)
+    ramoinput = read_yaml(filename)
+    occ_states = get_occupied_states(Electrum.PlanewaveWavefunction(ramoinput), ramoinput.emin, ramoinput.emax)
```

Comment:
There is a method that will directly construct an `OccupiedStates` from a `raMOInput`, so you can just call
```julia
occ_states = get_occupied_states(ramoinput)
```

Reply:
Resolved in [001b98c](https://github.com/xamberl/DFT-raMO/pull/39/commits/001b98c698962603e5cf15614e573730df3ca5ba)